plugins/kubernetes-react/api-report.md

```diff
@@ -0,0 +1,448 @@
+  // (undocumented)
+  kind: string;
+  // (undocumented)
+  object: T;
+  // (undocumented)
+  renderObject: (obj: T) => object;
+}
+
+// @public (undocumented)
+export const LinkErrorPanel: ({
+  cluster,
+  errorMessage,
+}: LinkErrorPanelProps) => React_2.JSX.Element;
+
+// @public (undocumented)
+export type LinkErrorPanelProps = {
+  cluster: ClusterAttributes;
+  errorMessage?: string;
+  children?: React_2.ReactNode;
+};
+
+// @public
+export const ManifestYaml: ({
+  object,
+}: ManifestYamlProps) => React_2.JSX.Element;
+
+// @public
+export interface ManifestYamlProps {
+  // (undocumented)
+  object: object;
+}
+
+// @public
+export const PendingPodContent: ({
+  pod,
+}: PendingPodContentProps) => React_2.JSX.Element;
+
+// @public
+export interface PendingPodContentProps {
+  // (undocumented)
+  pod: Pod;
+}
+
+// @public
+export interface PodAndErrors {
+  // (undocumented)
+  clusterName: string;
+  // (undocumented)
+  errors: DetectedError[];
+  // (undocumented)
+  pod: Pod;
+}
+
+// @public (undocumented)
+export type PodColumns = 'READY' | 'RESOURCE';
+
+// @public
+export const PodDrawer: ({
+  podAndErrors,
+  open,
+}: PodDrawerProps) => React_2.JSX.Element;
+
+// @public
+export interface PodDrawerProps {
+  // (undocumented)
+  open?: boolean;
+  // (undocumented)
+  podAndErrors: PodAndErrors;
+}
+
+// @public
+export const PodExecTerminal: (
+  props: PodExecTerminalProps,
+) => React_2.JSX.Element;
+
+// @public
+export const PodExecTerminalDialog: (
+  props: PodExecTerminalProps,
+) => React_2.JSX.Element;
+
+// @public
+export interface PodExecTerminalProps {
+  // (undocumented)
+  clusterName: string;
+  // (undocumented)
+  containerName: string;
+  // (undocumented)
+  podName: string;
+  // (undocumented)
+  podNamespace: string;
+}
+
+// @public
+export const PodLogs: React_2.FC<PodLogsProps>;
+
+// @public
+export const PodLogsDialog: ({
+  containerScope,
+}: PodLogsDialogProps) => React_2.JSX.Element;
+
+// @public
+export interface PodLogsDialogProps {
+  // (undocumented)
+  containerScope: ContainerScope;
+}
+
+// @public
+export interface PodLogsOptions {
+  // (undocumented)
+  containerScope: ContainerScope;
+  // (undocumented)
+  previous?: boolean;
+}
+
+// @public
+export interface PodLogsProps {
+  // (undocumented)
+  containerScope: ContainerScope;
+  // (undocumented)
+  previous?: boolean;
+}
+
+// @public
+export interface PodScope {
+  // (undocumented)
+  clusterName: string;
+  // (undocumented)
+  podName: string;
+  // (undocumented)
+  podNamespace: string;
+}
+
+// @public (undocumented)
+export const PodsTable: ({
+  pods,
+  extraColumns,
+}: PodsTablesProps) => React_2.JSX.Element;
+
+// @public (undocumented)
+export type PodsTablesProps = {
+  pods: Pod_2 | V1Pod[];
+  extraColumns?: PodColumns[];
+  children?: React_2.ReactNode;
+};
+
+// @public (undocumented)
+export const READY_COLUMNS: PodColumns;
+
+// @public (undocumented)
+export const RESOURCE_COLUMNS: PodColumns;
```

Comment:
Are these intended to be exported? This is the recommended pattern for exporting table column configuration: https://github.com/backstage/backstage/blob/49e92a6f85066e7ddd653a62783effce420b082b/plugins/github-deployments/src/components/GithubDeploymentsTable/GithubDeploymentsTable.tsx#L68

Can be a later refactor either way

Reply:
Yeah, lets' fix in a follow up